recipe/conda_build_config.yaml

```diff
@@ -70,27 +72,29 @@ flann:
 fontconfig:
   - 2.12
 freetype:
-  - 2.8                # [not ppc64le]
+  - 2.8.1              # [not ppc64le]
 gstreamer:
   - 1.12
 gst_plugins_base:
   - 1.12
+gdal:
+  - 2.2.*
 geos:
   - 3.6.2
 giflib:
   - 5.1
 glib:
-  - 2.53
+  - 2.55
```

Comment:
We are using [`glib 2.55`]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L79 ) currently. Switching to 2.53 will undoubtedly break things based on various symbols added and one removed over time.

ref: https://abi-laboratory.pro/tracker/timeline/glib/